crates/daphne-worker/src/aggregator/config.rs

```diff
@@ -0,0 +1,304 @@
+// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
```

Comment:
Believe it or not. Here and elsewhere.
```suggestion
// Copyright (c) 2025 Cloudflare, Inc. All rights reserved.
```